nixos/modules/services/web-apps/bentopdf.nix

```diff
@@ -0,0 +1,83 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.bentopdf;
+in
+{
+  options.services.bentopdf = {
+    enable = lib.mkEnableOption "bentopdf Privacy First PDF Toolkit";
+
+    package = lib.mkPackageOption pkgs "bentopdf" {
+      extraDescription = ''
+        To use the "simple mode" variant of bentopdf, which removes all socials, marketing and explanatory texts, set this option to `pkgs.bentopdf.overrideAttrs { SIMPLE_MODE = true; }`.
+      '';
+    };
+
+    domain = lib.mkOption {
+      type = lib.types.str;
+      default = "_";
+      description = "The nginx virtual host name to listen on. Set to `_` to forward all requests to bento.";
+    };
+
+    host = lib.mkOption {
+      type = lib.types.str;
+      default = "0.0.0.0";
+      description = "The host to listen on.";
+    };
+
+    port = lib.mkOption {
+      type = lib.types.port;
+      default = 3000;
+      description = "The port to listen on.";
+    };
+
+    openFirewall = lib.mkOption {
+      type = lib.types.bool;
+      default = true;
+      description = "Open the port nginx is listening on for bentopdf.";
+    };
```

Comment:
This is confusing. If nginx is normally configured it listens on 80 and 443. At least 80 is required for acme to properly work. We should stick to that